### PR TITLE
Show data discrepancies when in debug mode

### PR DESCRIPTION
**Pull request by arkadyan**
At the bottom of the Vehicle Properties panel.

View this by adding `?debug=1` to the URL.

Asana ticket: [Debug mode for data discrepancies](https://app.asana.com/0/1112935048846093/1128997245638196)

debug

### CODE REVIEW

**Comment by arkadyan** [comment]
Related to, but not directly affecting this change: it seems like Busloc might not ever be assigning a route ID or headway. If that is true, we likely will want to remove those properties for the list that are observed for discrepancies.

**Comment by skyqrose** [comment]
Are there any other fields that would be useful to have in the debug mode? Going through all the fields, these are the ones that might be useful and aren't displayed anywhere else:
* timestamp
* block_id
* run_id

File: assets/src/hooks/useVehicles.ts [hunk 1]
```diff
@@ -46,6 +56,8 @@ interface VehicleData {
   schedule_adherence_secs: number
   schedule_adherence_string: string
   scheduled_headway_secs: number
+  sources: string[]
```
Comment (skyqrose) [inline assets/src/hooks/useVehicles.ts]: Is this field used at all? Each discrepency has its own list of sources already.

**Comment by arkadyan** [comment]
@skyqrose At least for the moment I want to keep this very focused. We have developer tools that give us access to debug all the data, I want this to be a way for everyone to be able to explore differences in assignments.

File: lib/realtime/vehicle.ex [hunk 2]
```diff
@@ -69,6 +71,8 @@ defmodule Realtime.Vehicle do
     :operator_id,
     :operator_name,
     :run_id,
+    :sources,
+    :data_discrepancies,
```
Comment (skyqrose) [inline lib/realtime/vehicle.ex]: `[]` would be a reasonable and valid default for this field, so I don't think we need to require the field.

**Pull request merged**